docs/source/overview/api.rst

```diff
@@ -175,6 +175,53 @@ Note that a tag is automatically created for the resource with the name ``<names
                 * ``resource_name`` : name of the resource to be created in the Paraglider controller (note: this name will be scoped on cloud and namespace when stored)
                 * ``description``: JSON string describing the resource to be created (excluding networking details)
 
+Attach
+^^^^^^
+
+Attaches a resource to Paraglider according to the resource ID and the specified cloud. 
+Note that a tag is automatically created for the resource with the name ``<namespace>.<cloud>.<name>`` after attachment (where ``name`` is the resource name provided).
+
+.. tab-set::
+
+    .. tab-item:: CLI
+        :sync: cli
+
+        .. code-block:: shell
+            
+            glide resource attach <cloud> <resource_id>
+
+        Parameters:
+
+        * ``cloud``: name of the cloud to create the resource in
+        * ``resource_id`` : Id of the resource as specified by the cloud provider
+
+
+    .. tab-item:: REST
+        :sync: rest
+
+        .. tab-set::
+
+            .. tab-item:: POST
+
+                .. code-block:: shell
+
+                    POST /namespaces/{namespace}/clouds/{cloud}/resources
+
+                * Example request body:
+
+                .. code-block:: JSON
+
+                    {
+                        "id": "resource-id"
+                    }
+
+                Parameters:
+
+                * ``id``: Id of the resource as specified by the cloud provider
+    
+Note: Create resource and Attach resource share the same API endpoint. The difference between the two POST requests is the request body. If the request body contains a ``description``, the request is considered as a create resource request. Otherwise, it is considered as an attach resource request. 
```

Comment:
Let's make this into a callout

```suggestion
..note:
    Create resource and Attach resource share the same API endpoint. The difference between the two POST requests is the request body. If the request body contains a ``description``, the request is considered as a create resource request. Otherwise, it is considered as an attach resource request. 
```